crates/pop-contracts/Cargo.toml

```diff
@@ -40,3 +42,4 @@ pop-common = { path = "../pop-common", version = "0.5.0" }
 dirs.workspace = true
 mockito.workspace = true
 tokio-test.workspace = true
+hex = "0.4.3"
```

Comment:
This appears to be a mistake, please remove it.

Reply:
Thanks for catching this. It was used in a unit test and made its way in again. Resolved here: https://github.com/r0gue-io/pop-cli/pull/371/commits/89df5f6fedd9fe606352d19b4c156ff3aa8b3349